problems/1.types+operators.js

```diff
@@ -1,21 +1,28 @@
 /** Takes in an value and returns if it is a number
-* @param {any} val - The input value
-* @returns {boolean} - Is val a number
-*/
-function isNumber() {
-
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val a number
+ */
+function isNumber(val) {
+  if (typeof (val) === "number") {
```

Comment:
Can get away with just returning the condition here and no need to put the argument of typeof in ()